app/javascript/stylesheets/dashboard.scss

```diff
@@ -27,46 +27,10 @@
     }
   }
 
-  input[type="radio"].toggle + label {
-    font-size: smaller;
-    line-height: 1;
-    padding: 0.375rem 0.375rem;
-  }
-
-  .work-type-box {
-    @extend .col-md-3;
-    @extend .mb-5;
-
-    height: 2rem;
-  }
-
-  .work-type-button-text {
-    display: inline-block;
-    max-width: 60%;
-    padding-left: 1rem;
-    vertical-align: middle;
-  }
```

Comment:
Extracted all work type-related styles to a new file, outside of the `#dashboard` scope. This fixes a bug in how the modal renders when clicked from the collection show page.